gpkit/tests/t_nomials.py

```diff
@@ -221,7 +224,10 @@ def test_eq_ne(self):
         xu = Variable('x', units="ft")
         with SignomialsEnabled():
             self.assertEqual(x - x**2, -x**2 + x)
-            self.assertNotEqual(-x, -xu)
+            if gpkit.units:
+                self.assertNotEqual(-x, -xu)
+            else:  # units don't create inequality if they're disabled
+                self.assertEqual(-x, -xu)
```

Comment:
agree with the change, but really wondering if it's time to remove `disable_units`